apps/docs/content/components/Radio/index.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Radio
+description: Use the radio component when a user needs to select one option from a list.
+---
+
+import ComponentLayout from '../../../src/layouts/component-layout'
+export default ComponentLayout
+
+
+## Anatomy
+
+![An image displaying a Radio with a radio and label.](https://github.com/primer/brand/assets/131988618/1b494cdb-4991-4fc9-beac-feb02b296cb9)
+
+- Radio: A selection control.
+- Label: A description for the radio. 
+
+## Usage
+
+Use the radio component to enable users to select a single option from a list.
+
+For scenarios that allow for multiple selections, use the [checkbox](/components/Checkbox) component instead. Keep the radio label descriptive and concise: aim for no more than 3 words. Write labels in sentence case and without ending in punctuation.
+
+### Grouping radio components
+Use the `name` prop to group together related radio components in a list.
+
+If you're not building something custom, you should use the `Stack` and `FormControl` component to render a group of radio inputs.
```

Comment:
I think we could also simplify this by just saying that you could group related radio component in a list, using Stack and the FormControl components. I'd rather prefer to limit the prop exposure in the usage guidelines and leave it for the React section.